src/pages/Trainee/TraineeList.jsx

```diff
@@ -4,9 +4,13 @@ import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import { withStyles } from "@material-ui/core/styles";
 import AddDialog from "./Components/AddDialog/AddDialog";
+import EditDialog from "./Components/EditDialog/EditDialog";
+import RemoveDialog from "./Components/RemoveDialog/RemoveDialog";
 import trainee from "./data/trainee";
 import Table from "./Components/Table/Table";
 import moment from "moment";
+import EditIcon from "@material-ui/icons/Edit";
+import DeleteIcon from "@material-ui/icons/Delete";
```

Comment:
Please use single import statement

Reply:
It is not getting imported via a single import statement